src/spikeinterface/extractors/tests/test_neoextractors.py

```diff
@@ -278,8 +278,8 @@ class Spike2RecordingTest(RecordingCommonTestSuite, unittest.TestCase):
 
 
 @pytest.mark.skipif(
-    version.parse(platform.python_version()) >= version.parse("3.10"),
-    reason="Sonpy only testing with Python < 3.10!",
+    version.parse(platform.python_version()) >= version.parse("3.10") or platform.system() == "Darwin",
+    reason="Sonpy only testing with Python < 3.10 and not supported on macOS!",
```

Comment:
Is this an Arm issue?

Reply:
Yes, want to give it a shot to have something that excludes intel cpus from mac? I think they stopped selling two years ago only so many people might still have them.

That said, I did not want to allocate extra effort to something that won't work in a year anyway (see the python restriction)